### DISCUSSION TITLE
digest auth

### DISCUSSION THREAD

**Discussion by gilcu3** [patpadgett/corkscrew#10] [General]
Hi @patpadgett, a few years back I tried using corkscrew with my university proxy, and noticed that it didn't work because corkscrew does not support http digest auth. I could not find a way to contribute to the project at the time (the original webpage was down already), therefore created a [fork](https://github.com/gilcu3/corkscrew) with the patches.  Now I see you have moved the project to github :). Would you be willing to accept a pull request with this feature?

**Comment by patpadgett** [comment]
Sure.  Please submit a PR and I will take a look!

**Reply by gilcu3** [reply]
Then I will. I have one question though, is it ok to use an `md5` implementation such as [this](https://github.com/darkk/redsocks/blob/master/md5.h)? I am not totally sure whether the license is appropriate.
